meta/SaiSerialize.cpp

```diff
@@ -5440,3 +5500,79 @@ void sai_deserialize_redis_link_event_damping_aied_config(
     sai_deserialize_number(j["decay_half_life"], value.decay_half_life, false);
     sai_deserialize_number(j["flap_penalty"], value.flap_penalty, false);
 }
+
+/** @brief deserializes the stats capability list 
+*
+*   Iterates thru stat_enum_str and populate
+*   the stats_capability with respective
+*   stat_enum (String to Enum conversion).
+*   Also iterates thru stat_modes_str and populate
+*   the stats_capability with respective
+*   stat_modes (String to Enum conversion)
+* 
+*   @param stats_capability stats capability enum list
+*   @param stat_enum_str SAI stat enum list as string
+*   @param stat_modes_str SAI stat mode list as string
+*   @return Void
+*/
+void sai_deserialize_stats_capability_list(
+        _Inout_ sai_stat_capability_list_t *stats_capability,
+        _In_    const std::string& stat_enum_str,
+        _In_    const std::string& stat_modes_str)
```

Comment:
deserialization funciton does not follow convention, only 2 parameters string serialized and output paerameter

Reply:
Unlike the other deserialize functions, this stats_capability deserialization has stat_enum and stat_mode_enum as separate strings. Hence it was required to be handled this way. This function works fine in production. Please help , having it this way.